tests/acceptance/partial-move-test.js

```diff
@@ -0,0 +1,121 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+import '../factories/orders_package';
+import '../factories/designation';
+import '../factories/item';
+import '../factories/location';
+import '../factories/packages_location';
+import FactoryGuy from 'ember-data-factory-guy';
+import TestHelper from 'ember-data-factory-guy/factory-guy-test-helper';
+import { mockFindAll } from 'ember-data-factory-guy';
+
+var App, designation, item, orders_package, package_location;
+
+module('Acceptance: Partial move', {
+  beforeEach: function() {
+    App = startApp({}, 2);
+    TestHelper.setup();
+
+    designation = FactoryGuy.make("designation", { state: "closed" });
+    item = FactoryGuy.make("item", { state: "submitted", quantity: 5 });
+    package_location = FactoryGuy.make("packages_location", { quantity: 5 , item: item});
+    orders_package = FactoryGuy.make("orders_package", { state: "designated", quantity: 5, item: item, designation: designation });
+
+    mockFindAll('designation').returns({json: {designations: [designation.toJSON({includeId: true})]}});
+    visit("/items/" + item.id + "/partial_move");
```

Comment:
No @steveyken , as it is beforeEach block(although we can write some checks here but it will be unconventional). I've checked the page contents in test cases.